ckan/logic/schema.py

```diff
@@ -656,6 +656,8 @@ def default_activity_list_schema(
         ignore_missing, ignore_not_sysadmin, boolean_validator]
     schema['activity_types'] = [ignore_missing, list_of_strings]
     schema['exclude_activity_types'] = [ignore_missing, list_of_strings]
+    schema['before'] = [ignore_missing]
+    schema['after'] = [ignore_missing]
```

Comment:
These should be validated as timestamps, and as mentioned before if you want to check that both are not present do it here.

Reply:
Done [here](https://github.com/avdata99/ckan/commit/a3173276cbccd31bfe4c29d9490c40d5dee9cdbe)